tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -109,6 +115,24 @@ def example_plugin_info_non_scalar_feature():
     return info
 
 
+def example_plugin_info_mask_feature():
+    """plugin info for non-image feature"""
+    info = {
+        "method": compute_mask_plugin_feature,
+        "description": "This plugin will compute a mask feature",
+        "long description": "This mask feature is the mask "
+                            "data of RTDC",
+        "feature names": ["mask_feat"],
+        "feature labels": ["Mask or boolean Image"],
```

Comment:
This is intended as the "unit" of the feature. For example, for `area_um`, its feature label is "um" (micrometers). In this case there are not units, like here: https://dclab.readthedocs.io/en/stable/sec_av_notation.html#non-scalar-features